website/docs/api/textcategorizer.md

```diff
@@ -175,7 +175,7 @@ applied to the `Doc` in order. Both [`__call__`](/api/textcategorizer#call) and
 ## TextCategorizer.initialize {#initialize tag="method" new="3"}
 
 Initialize the component for training. `get_examples` should be a function that
-returns an iterable of [`Example`](/api/example) objects. The data examples are
+returns an iterable of [`Example`](/api/example) objects. **At least one annotated example should be supplied.** The data examples are
```

Comment:
You should run `prettier` to fix where the newlines are. (Fine to do it after review, since it makes diffs less clear.)